input/cql/AHAOverall.cql

```diff
@@ -35,7 +35,9 @@ context Patient
                 * nursing facility visits (http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.464.1003.101.12.1012)
                 * office visits (http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.464.1003.101.12.1001)
                 * outpatient consolutations (http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.3.464.1003.101.12.1008)
-*/   
+
+@QICore6Uplift: Update Condition to ConditionProblemsHealthConcerns, and consider adding 'type in' for Encounters
```

Comment:
I was looking at the "Heart Failure" condition, which I believe is more of a long-term condition and hence, a ConditionProblemsHealthConcerns. But, it does not specify whether it's acute or chronic... Thoughts? @jbyrdevans

Reply:
If the FHIR query doesn't restrict to a particular category, then I would suggest a union of [ConditionProblemsHealthConcerns] and [ConditionEncounterDiagnosis].

If we still feel that this isn't the intention of the measure, then we should seek clinical confirmation/update from the AIR team to target the appropriate Condition instead of the union of the two.